docs/releases/2.0.txt

```diff
@@ -97,6 +97,12 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* ...
+
+:mod:`django.contrib.admindocs`
```

Comment:
Undo this unrelated change... (Likely caused by merging master into the branch instead of rebasing.)